src/code/z_pause.c

```diff
@@ -1,19 +1,29 @@
+/**
```

Comment:
I think I'd like to start this off with `File:` and `Description:` lines like in overlay files... we do need to discuss how we'd like to format them (should the initial description be a synopsis and then a fuller description given under a different header, for example?

Reply:
Added `File:` and `Description:` lines to the header.